docs/core-concepts/core-sdk/sdk-overview.md

```diff
@@ -0,0 +1,6 @@
+# 🔧 .NET SDK Overview 
```

Comment:
What will go in this doc eventually? Describe what comes in the SDK and how to acquire it?